collect-tips.rst

```diff
@@ -0,0 +1,7 @@
+**************************
+Best Practices and Tips
+**************************
+
+- Specifying the GPS requirements can help users decide the accuracy of location they send and hence help save battery. For more information on enabling GPS services and its relation with battery life visit `this site <https://support.google.com/nexus/answer/3467281?hl=en>`_.
+
+- Rather than using encrypted forms, enabling Android-level data encryption can secure the data more efficiently. Encrypting the phone stores the data in an unreadable and scrambled form as long as it's locked. Whenever you enter your password, the data becomes readable. To encrypt your device, follow the instructions given `here <https://www.howtogeek.com/141953/how-to-encrypt-your-android-phone-and-why-you-might-want-to/>`_. 
```

Comment:
Using encrypted forms and encrypting the device solve different problems so they should not be presented as alternatives. Encrypting the form ensures that data is safe even sent over an HTTP connection and that it can be stored on a server that is not secure. Encrypting the phone is a good additional precaution because forms are not encrypted until they are finalized.

I would prefer https://support.google.com/nexus/answer/2844831?hl=en as the resource for using device-level encryption.

Reply:
@lognaturel Oh, I didn't know this. Guess I'll read more about it and update the contents accordingly. Thank you for pointing this out :)

Reply:
Clearly it needs to be documented!